### PR TITLE
Jmvera255 FAQs and Modules Updates (formerly patch 9)

### PR DESCRIPTION
**Pull request by jmvera255**

### CODE REVIEW

File: training/resources/frequently-asked-questions-faq-.md [hunk 1]
```diff
@@ -1,60 +1,58 @@
...
-
-**How many jobs can I keep idle in the queue?**
+Jobs are submitted via HTCondor scheduler. Please see our [QuickStart](http://support.opensciencegrid.org/support/solutions/articles/5000633410-osg-connect-quickstart) guide for more details on submitting and managing jobs.
 
-The preferred number of idle jobs is to not exceed 10k. If you have much more jobs than that, we recommend that you use HTCondor DAGMan to control it. Stick all the jobs in the DAG, and then submit with: condor_submit_dag -maxidle 10000 ...  DAGMan will then maintain the 10k idle jobs for you.
+**How many jobs can I have in the queue?**
 
-##Data Storage and Transfer
+The number of jobs that are submitted to the queue by any one user should not exceed 10,000. If you have more jobs than that, we ask that you include the following statement in your submit file: 
+`maxidle 2000`
```
Comment (ChristinaLK) [inline training/resources/frequently-asked-questions-faq-.md]:
```suggestion
`max_idle  = 2000`  

```

File: training/resources/frequently-asked-questions-faq-.md [hunk 2]
```diff
@@ -1,60 +1,58 @@
...
+   
+## Running Jobs
+   
+**What type of computation is a good match or NOT a good match for OSG Connect?**
 
-##Running Jobs
-**What type of computation is a good match or NOT a good match on OSG Connect?**
-
-It is important for the user to carefully check the computational requirements of his or her workflow. Please refer to ["this section"](http://support.opensciencegrid.org/solution/categories/5000131843/folders/5000209523/articles/5000632058-is-high-throughput-computing-for-you-) for more details on how to determine if your computation matches up well with OSG Connect's model.
+It is to carefully check the computational requirements of your workflow. Please see our ["Is the Open Science Grid for You?"](http://support.opensciencegrid.org/solution/categories/5000131843/folders/5000209523/articles/5000632058-is-high-throughput-computing-for-you-) page for more details on how to determine if your work matches up well with OSG Connect's model.
```
Comment (ChristinaLK) [inline training/resources/frequently-asked-questions-faq-.md]:
```suggestion
It is important to carefully check the computational requirements of your workflow. Please see our ["Is the Open Science Grid for You?"](http://support.opensciencegrid.org/solution/categories/5000131843/folders/5000209523/articles/5000632058-is-high-throughput-computing-for-you-) page for more details on how to determine if your work matches up well with OSG Connect's model.

```

File: training/resources/frequently-asked-questions-faq-.md [hunk 3]
```diff
@@ -1,60 +1,58 @@
...
 
 Please follow the steps outlined in the [Sign Up process](http://osgconnect.net/signup). 
 
-##Software
+## Software
+   
 **What software packages are available?**
 
-The available software are listed [here](http://support.opensciencegrid.org/solution/categories/5000160799/folders/5000260921/articles/5000634397-software-modules-catalog).   In general, we support most popular software that fits the distributed high thoroughput model and can add software upon request.
+The available software are listed [here](https://support.opensciencegrid.org/support/solutions/articles/12000048518). In general, we support most software that fits the distributed high thoroughput computing model. Additionally, we can add software upon request.
```
Comment (ChristinaLK) [inline training/resources/frequently-asked-questions-faq-.md]:
```suggestion
In general, we support most software that fits the distributed high throughput computing model. Users are encouraged to download and install their own software. For some software, we support distributed software modules listed [here](https://support.opensciencegrid.org/support/solutions/articles/12000048518). Software can be added to the modules upon request. 

```

File: training/resources/frequently-asked-questions-faq-.md (hunk 1 above)
Comment (jmvera255) [reply training/resources/frequently-asked-questions-faq-.md]: nice catch!

File: training/resources/frequently-asked-questions-faq-.md [hunk 4]
```diff
@@ -1,109 +1,112 @@
...
+
+## Data Storage and Transfer
+   
 **What is the best way to process large volume of data?**
 
-Use the Stash data system to store and process large volumes of data. Please refer the section [Data Solutions](http://support.opensciencegrid.org/support/solutions/folders/5000262152) for more details. 
+Use the Stash data system to stage large volumes of data. Please refer the section [Data Solutions](http://support.opensciencegrid.org/support/solutions/folders/5000262152) for more details. 
  
-**How do I transfer data from Stash to my system and vice versa?**
+**How do I transfer my data to and from OSG Connect?
```
Comment (ChristinaLK) [inline training/resources/frequently-asked-questions-faq-.md:61]:
```suggestion
**How do I transfer my data to and from OSG Connect?**

```

**Comment by jmvera255** [comment]
Sorry, not actually ready to merge. Should I close the PR and make the request again later?

File: training/resources/frequently-asked-questions-faq-.md (hunk 4 above)
Comment (jmvera255) [reply training/resources/frequently-asked-questions-faq-.md]: fixed in commit 4269b61

**Comment by ChristinaLK** [comment]
No, that's fine - just leave it up and keep adding changes.

**Comment by jmvera255** [comment]
OK, both the FAQs and Modules updates have been completed and are ready for final review, sorry for all the back and forth... @ChristinaLK @rynge @lmichael107

**Pull request merged**